sdk/agrifood/agrifood-farming-rest/src/clientDefinitions.ts

```diff
@@ -0,0 +1,1984 @@
+   *
+   *
+   */
+  get(options?: CropsListParameters): Promise<CropsList200Response | CropsListdefaultResponse>;
+}
+
+export interface CropsGet {
+  /**
+   * Get a specified crop resource.
+   *
+   *
+   */
+  get(options?: CropsGetParameters): Promise<CropsGet200Response | CropsGetdefaultResponse>;
+  /**
+   * Create or update a crop resource.
+   *
+   * 
+###### Note:
+   * 
+1. The **‘contentType’** in the request header should be **'application/merge-patch+json'**.
+   * 2. 'Id' of a resource can only contain alphanumeric characters, '-', '_', '~' or '.', and must begin and end with an alphanumeric character. Minimum length of an ‘Id’ is 2 characters and maximum length is 50 characters.
+   * 3. 'Key' and 'value' in 'Properties' can have a maximum of 50 and 150 characters respectively.
+   *
+   *
+   */
+  patch(
+    options?: CropsCreateOrUpdateParameters
+  ): Promise<
+    | CropsCreateOrUpdate200Response
+    | CropsCreateOrUpdate201Response
+    | CropsCreateOrUpdatedefaultResponse
+  >;
+  /**
+   * Delete a specified crop resource.
+   *
+   *
+   */
+  delete(
+    options?: CropsDeleteParameters
+  ): Promise<CropsDelete204Response | CropsDeletedefaultResponse>;
+}
+
+export interface CropVarietiesListByCropId {
+  /**
+   * Get a paginated list of crop variety resources for a given crop and search criteria.
+   *
+   * 
+###### Note: 
+   * 1. Users will get paginated response with each page containing a link to the next page. Users are expected to iterate over all the pages using ‘nextLink’. The search comes to an end when ‘nextLink’ is null.
+   *
+   *
+   */
+  get(
+    options?: CropVarietiesListByCropIdParameters
+  ): Promise<CropVarietiesListByCropId200Response | CropVarietiesListByCropIddefaultResponse>;
+}
+
+export interface CropVarietiesList {
+  /**
+   * Get a paginated list of crop variety resources across all the crops for a given search criteria.
+   *
+   * 
+###### Note:
+   * 1. This API does a serial search of the database for the given search criteria. If the crop Id is available, it is strongly recommended to use **‘List by Crop Id’** API as it is performant.
+   * 2. Users will get paginated response with each page containing a link to the next page. Few pages may have **no items (zero results)**. Users are expected to iterate over all the pages using ‘nextLink’. The search comes to an end when ‘nextLink’ is null.
+   *
+   *
+   */
+  get(
+    options?: CropVarietiesListParameters
+  ): Promise<CropVarietiesList200Response | CropVarietiesListdefaultResponse>;
+}
+
+export interface CropVarietiesGet {
+  /**
+   * Get a specified crop variety resource for a given crop.
+   *
+   *
+   */
+  get(
+    options?: CropVarietiesGetParameters
+  ): Promise<CropVarietiesGet200Response | CropVarietiesGetdefaultResponse>;
+  /**
+   * Create or update a crop variety resource.
+   *
+   * 
+###### Note:
+   * 
+1. The **‘contentType’** in the request header should be **'application/merge-patch+json'**.
```

Comment:
it feels strange that we are doing `**'contentType'**` - especially since the single quotes look like smart quotes? 

Also the header is "Content-Type" not "contentType"... but I am guessing this remark comes from swagger?

Reply:
Yeah this is coming from the Swagger